server.js

```diff
@@ -2,25 +2,105 @@ import express from "express";
 import cors from "cors";
 import mongoose from "mongoose";
 
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/projectHappyThoughtsAPI";
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
 mongoose.Promise = Promise;
 
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
+// Defines the port the app will run on.
 const port = process.env.PORT || 8080;
 const app = express();
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
+// Middleware set up for error msg like DB down.
+app.use((req, res, next) => {
+  if (mongoose.connection.readyState === 1) {
+    next()
+  } else {
+    res.status(503).json({ error: 'Service unavailable' })
+  }
+}) 
+
+// Start page
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+   res.send({
+    responseMassage: "Hello Everybody and welcome, This is a backend built for Frontend project Happy Thoughts from w7!",
+    guide: "To access the frontend use netlify Link: https://happy-thought-nina-berggren.netlify.app/ ",
+    Routes:[{
+        "/thoughts": "send GET request to see all thoughts, send POST request to create a new Happy Thought.",
+        "/thoughts/:id/like": "Updates and stor how many likes (hearths) a thought gets.",
+      }],
```

Comment:
Love this in the base path! Makes it easier to understand what the API is about.